app/routes/useApi.jsx

```diff
@@ -150,15 +204,16 @@ export default function useApi() {
     */
     const addIngredientsToPantry = async (ingredients) => {
         const body = { ingredients };
-        return await apiCall('/pantry/addIngredients', 'POST', body);
+        body.sign = 1;
```

Comment:
Is the `body.sign = -1` a change agreed-to with the backend team?

Reply:
En add Ingredient es 1 y en remove es -1